content/docs/csidriver/release/unity.md

```diff
@@ -3,17 +3,17 @@ title: Unity
 description: Release notes for Unity CSI driver
 ---
 
-## Release Notes - CSI Unity v2.0.0
+## Release Notes - CSI Unity v2.1.0
 
 ### New Features/Changes
 
-- Added support for Kubernetes v1.22.
-- Added support for Openshift 4.8.
-- Added the ability to change log level and log format of CSI driver and change them dynamically.
-- Added the ability to configure kubelet directory path.
-- Added the ability to enable/disable installation of resizer sidecar with driver installation.
-- Added the ability to enable/disable installation of snapshotter sidecar with driver installation.
-- Added support for consistent config parameters across CSI drivers.
+- Added support for OpenShift v4.9.
+- Added support for CSI spec 1.5.
+- Added support for new access modes in CSI Spec 1.5.
+- Added support for PV/PVC metrics.
```

Comment:
@shanmydell "volume health monitoring" covers pvc metrics part. So we can remove the following statements here
- Added support for PV/PVC metrics.
- Added NFS support for PVC metrics in controller.

Reply:
Well perhaps we just simplify to 'Added support for PV/PVC metrics' as even without monitor enabled there are still metrics gathering, the monitor just adds to that.